test/ui-pytest/gui/components/authenticate_popup.py

```diff
@@ -0,0 +1,26 @@
+import allure
+
+from gui.elements.qt.button import Button
+from gui.elements.qt.text_edit import TextEdit
+from gui.elements.qt.object import QObject
+
+
+
+class AuthenticatePopup(QObject):
```

Comment:
Two lines before classes
Please use hotkey to refactor your code:
https://www.jetbrains.com/help/pycharm/reformat-file-dialog.html#:~:text=The%20dialog%20appears%20when%20you,of%20the%20specified%20scope%20automatically.

Reply:
Fixed. And thank you @StateOf-Vlado, it's very useful